packages/blob/src/index.js

```diff
@@ -3,6 +3,9 @@
  */
 const { createObjectURL, revokeObjectURL } = window.URL;
 
+/**
+ * @type {{[key: string]: File|undefined}}
```

Comment:
Initially, my editor ignored the `| undefined` part of this type and the following one. Adding explicit `strictNullChecks` to the tsconfig restored it.